utils/scale-test

```diff
@@ -38,20 +38,23 @@ def has_debuginfo(swiftc):
 
 
 def write_input_file(args, ast, d, n):
-    ifile = os.path.join(d, "in%d.swift" % n)
-    with open(ifile,'w+') as f:
+    fname = "in%d.swift" % n
+    pathname = os.path.join(d, fname)
+    with open(pathname,'w+') as f:
```

Comment:
nit-pick: You should run the `flake8` linter over this code. I think this line will produce a warning, because it's missing a space after the comma.